Azure Flexible DB for MySQL Document (#4665)
SumoLogic/sumologic-documentation: docs/integrations/microsoft-azure/azure-database-for-mysql.md

* **MySQL Slow Query Logs**. In Azure Database for MySQL flexible server, the slow query log is available to users to configure and access. Slow query logs are disabled by default and can be enabled to assist with identifying performance bottlenecks during troubleshooting. To learn more about the different log types and schemas collected for Azure Database for MySQL, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/mysql/flexible-server/concepts-slow-query-logs#access-slow-query-logs).
* **MySQL Error Logs**. In Azure Database for MySQL flexible server, the error log is available to users to configure and access. Error logs in MySQL gather diagnostic messages during server startup and shutdown and while the server is running, to provide information that can help proactive troubleshooting. For more information about the MySQL error log, see the [Error Log Documentation](https://learn.microsoft.com/en-us/azure/mysql/flexible-server/concepts-error-logs).
   :::note

>    MySQL Error Logs are under in a preview phase. Error logs are available under Server Logs only, and error logs can't be emitted to Azure diagnostic logs. You need to download logs from the **Server Logs > Error Logs** menu send them to a Sumo Logic HTTP collector endpoint via curl request.

Reviewer: ```suggestion
   MySQL error logs are currently in a preview phase and are only available under Server Logs. These logs cannot be emitted to Azure diagnostic logs directly. To access the error logs, navigate to **Server Logs > Error Logs**, download them, and then send the logs to a Sumo Logic HTTP collector endpoint using a `curl` request.
```

>    MySQL Error Logs are currently in a preview phase and are only available under Server Logs. These logs cannot be emitted to Azure diagnostic logs directly. To access the error logs, navigate to **Server Logs > Error Logs**, download them, and then send the logs to a Sumo Logic HTTP collector endpoint using a `curl` request.

Reviewer 2: ```suggestion
then [upload the logs](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/upload-logs) to a Sumo Logic HTTP source endpoint.
```

1. To set up the Azure Event Hubs source in Sumo Logic, refer to the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
1. To create the diagnostic settings in Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/data-factory/monitor-configure-diagnostics). Perform the steps below for each Azure Redis cache account that you want to monitor.
   1. Choose **Stream to an event hub** as the destination.
   1. Select `allLogs`.
   1. Use the Event Hub namespace and Event Hub name configured in the previous step in the destination details section. You can use the default policy `RootManageSharedAccessKey` as the policy name.
1. Tag the location field in the source with right location value. <br/><img src={useBaseUrl('img/integrations/microsoft-azure/Azure-Storage-Tag-Location.png')} alt="Azure Redis Cache Tag Location" style={{border: '1px solid gray'}} width="400" /> <br/><img src={useBaseUrl('img/send-data/azureflexible-mysqlserver-logs.png')} alt="Azure flexible mysql server logs" style={{border: '1px solid gray'}} width="800" /> 

> #### Activity Logs

Reviewer 2: Need to add section for 
Enabling error logs (along with the required query parameters)
Enabling Slow query logs (along with the required query parameters)
Enabling Audit  logs (along with the required query parameters)

Also is there any extra step required for enabling any health related logs and security (cloud defender ) related logs?

> To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit). Do not perform this step if you are already collecting activity logs for a subscription.

Reviewer: ```suggestion
To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit). Skip this step if you are already collecting activity logs for a subscription.
```

:::note
Since this source contains logs from multiple regions, ensure that you do not tag this source with the location tag.
:::

## Installing the Azure Flexible Database for Mysql app

Now that you have set up data collection, install the Azure Load Balancer Sumo Logic app to use the pre-configured dashboards that provide visibility into your environment for real-time analysis of overall usage.

import AppInstallNoDataSourceV2 from '../../reuse/apps/app-install-index-apps-v2.md';

<AppInstallNoDataSourceV2/>

## Viewing the  Flexible Database for Mysql dashboards

import ViewDashboards from '../../reuse/apps/view-dashboards.md';

<ViewDashboards/>

### Error Logs

> The **Azure Database for Mysql - Error Logs**

Reviewer: I'd eliminate the `The` in these sections since it's not a full sentence.

Reviewer 3: We need to add more information to the dashboard description. cc: @sachin-sumologic
